Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ruby:3.2.2
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev libvips
```

Comment:
`apt-get clean && rm -rf /var/lib/apt/lists/*` will reduce the image size slightly.

And, I personally prefer if the Dockefile is divided into several blocks to increase readability. Like this, 

```dockerfile
FROM ruby:3.2.2

ENV LANG=ja_JP.UTF-8
ENV TZ=Asia/Tokyo

WORKDIR /app

RUN apt-get update -qq && apt-get install -y \
    build-essential \
    libpq-dev \
    libvips \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*

COPY Gemfile Gemfile.lock /app/

RUN bundle install
```

Reply:
Fixed with 453707e !